src/Illuminate/Database/Console/DumpCommand.php

```diff
@@ -73,7 +73,7 @@ protected function schemaState(Connection $connection)
      */
     protected function path(Connection $connection)
     {
-        return tap($this->option('path') ?: database_path('schema/'.$connection->getName().'-schema.sql'), function ($path) {
+        return tap($this->option('path') ?: database_path('schema/'.$connection->getName().'-schema.dump'), function ($path) {
```

Comment:
Why?

Reply:
The result obtained with ** pg_dump -Fc ** is not SQL therefore not readable like classic SQL ...